server/test/integration/vitalsTest/handleHeartbeatTest.js

```diff
@@ -96,6 +96,45 @@ async function lowBatteryHeartbeat(coreId) {
   }
 }
 
+async function doorFallOffHeartbeat(coreId) {
```

Comment:
similarly, I think all instances of "doorfallOff" should be reworded to "doorInactivty"

Reply:
fixed in latest push!